src/ComputeEngine.java

```diff
@@ -1,4 +1,6 @@
+import java.util.List;
 public interface ComputeEngine {
   //inputs raw data and fetches results/error code
   ComputeResults compute(RawData data);
+  List<Integer> compute(List<Integer> data);
```

Comment:
good idea, since we are using Lists instead of a unspecified type this will be useful as an option. Perhaps we can also make them generic types in the future